streams/lt.m3u

```diff
@@ -19,3 +19,17 @@ https://radijas.lrt.lt/radijas/master.m3u8
 https://radijas.lrt.lt/klasika/master.m3u8
 #EXTINF:-1 tvg-id="LRTOpus.lt",LRT Opus (1080p)
 https://radijas.lrt.lt/opus/master.m3u8
+#EXTINF:-1 tvg-id="LNK.lt",LNK (1080p)
+https://live.lnk.lt/lnk_live/smil:lnk_live.smil/playlist.m3u8?tokenstarttime=0&tokenendtime=1683259096&tokenhash=iOBCxbO8M6HzvpKYFaBxWNZcCdDiYfMk2sVLDu_Jx6MHIvOIOUESXwJRGYHau7Q3j70CiTayc3axjxix4WSktA==
+#EXTINF:-1 tvg-id="BTV.lt",BTV (1080p)
+https://live.lnk.lt/lnk_live/smil:btv_live.smil/playlist.m3u8?tokenstarttime=0&tokenendtime=1683259326&tokenhash=Z-qCE9n27ryLgi3p8uaIqwObuY5Eym1DHVoIpfmGKW6VjN-sjqsiboIh34wlyt3RVe49Vrx64giMMAj0TCJqeQ==
+#EXTINF:-1 tvg-id="LRTTV.lt",LRT HD (1080p)
+https://lrthd.lrt.lt/secure/lrt_hd/TK8DdmrYi1ahTMKCxVOdFg/1683237332/master.m3u8
```

Comment:
offline
<img width="619" alt="image" src="https://user-images.githubusercontent.com/7253922/236336449-fd8b72d1-87a0-4d34-aed6-9a4babc95828.png">

Reply:
![image](https://user-images.githubusercontent.com/132615385/236337380-5bb03bdc-0523-45c9-bb92-8991ea111fb6.png)
Works for me in VLC. Is it possible it's geo-blocked, but showing up as offline in the test? If so, I'll add the geo-blocked label.

Reply:
Perhaps you are right, the links are indeed geo-blocked.

Reply:
https://www.lrt.lt/servisai/stream_url/live/get_live_url.php?channel=LTV1 and works from a French IP.

You can scrape the link from there.

Reply:
Tried the link in Streamtest, returns offline. My guess is that it's either geo-blocked to europe or the test just can't see it